js/views/PageNav.js

```diff
@@ -207,13 +209,21 @@ export default class extends View {
 
   render() {
     let avatarHash = '';
-    if (isHiRez() && app.profile && app.profile.avatarHashes.small) {
-      avatarHash = app.profile.avatarHashes.small;
-    } else if (app.profile && app.profile.avatarHashes.tiny) {
-      avatarHash = app.profile.avatarHashes.tiny;
+
+    if (app.profile && isHiRez()) {
```

Comment:
This is fixed in the more-store branch, which is coming later. The reason it's not here is because at the time I wrote this, the branch with isHiRez function was not merge into Master yet.